packages/react-ui-core/src/Form/Form.js

```diff
@@ -40,22 +40,30 @@ export default class Form extends Component {
      * Serializes form data when true.
      */
     serialize: PropTypes.bool,
+
+    /**
+     * Serialize options when serialize is on
+     */
+    serializeOptions: PropTypes.object,
   }
 
   static defaultProps = {
     theme: {},
     action: '#',
     method: 'POST',
     noValidate: true,
+    serializeOptions: {
+      hash: true,
```

Comment:
what does this do

Reply:
Which part? The hash true was there before, but without adding `empty: true` it wasn't allowing us to override redux properly. This just gives you the option to define how you want to serialize.